core/cmd/app.go

```diff
@@ -121,28 +128,35 @@ func NewApp(client *Client) *cli.App {
 			if err := opts.ParseTOML(configTOML, secretsTOML); err != nil {
 				return err
 			}
-			lggrCfg := logger.Config{
-				LogLevel:       zapcore.Level(*opts.Config.Log.Level),
-				Dir:            *opts.Config.Log.File.Dir,
-				JsonConsole:    *opts.Config.Log.JSONConsole,
-				UnixTS:         *opts.Config.Log.UnixTS,
-				FileMaxSizeMB:  int(*opts.Config.Log.File.MaxSize / utils.MB),
-				FileMaxAgeDays: int(*opts.Config.Log.File.MaxAgeDays),
-				FileMaxBackups: int(*opts.Config.Log.File.MaxBackups),
-			}
-			client.Logger, client.CloseLogger = lggrCfg.New()
```

Comment:
We can't construct the logger directly from the parsed config since it doesn't have the default values filled in yet.